tests/Networking/NexusMods.Networking.HttpDownloader.Tests/LocalHttpServer.cs

```diff
@@ -9,14 +12,34 @@ public class LocalHttpServer : IDisposable
     private readonly HttpListener _listener;
     private readonly string _prefix;
 
+    public byte[] LargeData { get; set; }
+    public Hash LargeDataHash { get; set; }
+
+
     public LocalHttpServer(ILogger<LocalHttpServer> logger)
     {
         _logger = logger;
         (_listener, _prefix) = CreateNewListener();
         _listener.Start();
+
+        LargeData = GenerateLargeData();
+        LargeDataHash = LargeData.AsSpan().XxHash64();
+
         StartLoop();
     }
 
+
+
+    private byte[] GenerateLargeData()
+    {
+        var data = new byte[512 * 1024 * 1024];
+        var seed = Random.Shared.Next(0, 255);
```

Comment:
NextBytes is extremely slow, and I think I forgot to make a comment to that effect

Reply:
Ah, I see. No issue then.

Kind of surprised though, I'd expect it to use `System.Private.CoreLib.Random.XoshiroImpl` (available since .NET 6) under the hood, and the logic for that looked alright. If perf is an issue, can always use the full register :wink: 

Anyway, comment and close.